brevent/src/main/res/values-es/strings.xml

```diff
@@ -121,13 +121,13 @@
     <string name="brevent_auto">Auto Brevent</string>
     <string name="brevent_auto_label">Actualizar la lista de Brevent cuando las aplicaciones se instalen o desinstalen.</string>
     <string name="brevent_method">Método Brevent</string>
-    <string name="brevent_method_standby_forcestop">Standby, después forzar detener</string>
+    <string name="brevent_method_standby_forcestop">Standby,después forzar detener</string>
```

Comment:
I think, there should be a space.

Reply:
Without the space or without the point  could enter a line.  That's why I suggest change. Thanks in advance.

Reply:
Please make a space. There are other screen-width devices.